src/Controllers/ImageController.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace JustBetter\GlideDirective\Controllers;
+
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+use Illuminate\Routing\Controller;
+use Illuminate\Support\Str;
+use JustBetter\GlideDirective\Responsive;
+use Statamic\Contracts\Assets\Asset;
+use Statamic\Contracts\Imaging\ImageManipulator;
+use Statamic\Facades\Asset as AssetFacade;
+use Statamic\Facades\Image;
+use Statamic\Imaging\GlideImageManipulator;
+
+class ImageController extends Controller
+{
+    public function getPreset(Request $request, string $preset, string $fit, string $signature, string $file, string $format): Response
```

Comment:
You're not using the signature to validate the request yet 😉 
https://github.com/thephpleague/glide/blob/6ce7f6b337c61c05c97d664f6706e58918126f14/src/Signatures/Signature.php#L45